secp256k1jni/src/main/java/org/bitcoin/Secp256k1Context.java

```diff
@@ -46,7 +46,14 @@
   }
 
   public static boolean isEnabled() {
```

Comment:
Let's add a javadoc to the top of this what the env var is for, and why it is useful